src/Api/Auth/Models/Public/AssociationWithPermissionsBaseModel.cs

```diff
@@ -15,4 +15,9 @@ public abstract class AssociationWithPermissionsBaseModel
     /// </summary>
     [Required]
     public bool? ReadOnly { get; set; }
+    /// <summary>
+    /// When true, the hide passwords permission will not allow the user or group to view passwords.
```

Comment:
This is not a particularly secure feature, I recommend adding the disclaimer from our help docs:

```suggestion
    /// When true, the hide passwords permission will not allow the user or group to view passwords.
    /// This prevents easy copy-and-paste of hidden items, however it may not completely prevent user access.
```